src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -623,10 +635,6 @@ bool fx_muxer_t::resolve_sdk_dotnet_path(const pal::string_t& own_dir, pal::stri
             }
         }
     }
```

Comment:
Instead of removing the error message, you may want to emit a warning for logging purposes.